app/mapper/ReservationMapper.py

```diff
@@ -31,10 +33,10 @@ def findAll():
         return
     else:
         for index, r in enumerate(result):
-            room = RoomMapper.find(result[index][1])
-            holder = UserMapper.find(result[index][3])
-            timeslot = TimeslotMapper.find(result[index][4])
-            reservation = Reservation(room, holder, timeslot, result[index][2], result[index][0])
+            room = RoomMapper.find(result[0][1])
```

Comment:
This is a bug that was reintroduced in your PR  (returns a list but only contains copies of the first element). No worries, I will fix in my next PR.